build/lib/extensions.ts

```diff
@@ -278,7 +278,6 @@ const externalExtensions = [
 	'azcli',
 	'azuremonitor',
 	'cms',
-	'context-builder',
```

Comment:
You need to rebuild the build folder and commit the change to the js version of this file too

Reply:
There seems to be a synchronization error between the changes in my branch and this PR not seeing them: 
![image](https://github.com/microsoft/azuredatastudio/assets/87730006/63baf3e1-eb81-478e-9edb-149dbf7ea246)

Reply:
Made new PR here with the compiled build directory output: https://github.com/microsoft/azuredatastudio/pull/24156